buildtools/maven/maven.go

```diff
@@ -171,6 +172,14 @@ func ParseDependencyTree(stdin string) (graph.Deps, error) {
 		if strings.HasPrefix(line, "Download") {
 			continue
 		}
+
+		if strings.Contains(line, "--------<") && len(filteredLines) > 0 {
```

Comment:
Does `--------<` always work as an indicator? I believe the formatting of the output is `--< whatever the module name is >--` and the longer the module name the shorter the surrounding `--` markers.

Reply:
You're correct that the longer the module name is the shorter the surrounding markers are. This solution worked for the 3 or 4 examples that each had 10-100 modules. It might be worth creating my own maven module with a larger name.

Reply:
What if we replace this with a prefix regexp that matches something like `^\[INFO\] -+<`?